### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by dipen2000**
Created avatar and badge components from the component library , please review my PR.

### CODE REVIEW

File: CSS/documentation.css [hunk 1]
```diff
@@ -119,14 +144,37 @@ main{
...
+.component-display{
+    /* border : 3px solid red; */
+    display : flex;
+    flex-direction: row;
+    align-items : center;
+    justify-content : center;
+    flex-wrap: wrap;
+    gap : 2rem;
+    padding : 1.5rem;
+    background-color : rgba(102, 51, 153, 0.356);
```
Comment (dhimanaishwarya) [inline CSS/documentation.css:158]: use variable for color everywhere

File: CSS/documentation.css [hunk 2]
```diff
@@ -119,14 +144,37 @@ main{
...
+    border-top-left-radius: 15px;
+    border-top-right-radius: 15px;
+}
 
+.component-code-section{
+    /* border : 3px solid red; */
+    padding : 8px;
+    border-bottom-left-radius: 15px;
+    border-bottom-right-radius: 15px;
```
Comment (dhimanaishwarya) [inline CSS/documentation.css:167]: above 10px you can use rem unit

File: components/avatar/avatar.html [hunk 3]
```diff
@@ -65,43 +68,85 @@ <h3>Components</h3>
                             </div>
                         </div>
                     </aside>
-                    <div class="content-main-section"></div>
+                    <div class="content-main-section">
+                        <h1>Avatar</h1>
+                        <p class="mb-2">Avatars can be used for user profile picture.</p>
+                        <div class="divider mb-2"></div>
+                        <h2 class="mb-2" id="same sized avatars">Same sized avatars</h2>
+                        <p class="mb-2">These are all the rounded same sized buttons</p>
```
Comment (dhimanaishwarya) [inline components/avatar/avatar.html:76]: documentation to be corrected here

File: CSS/documentation.css [hunk 4]
```diff
@@ -68,23 +69,47 @@ main{
...
     grid-area: main-content;
 }
 
+.content-main-section h1{
+    font-size: 4rem;
+    font-family: var(--component-heading-font);
+}
+
+.content-main-section h2{
+    /* font-size: 4rem; */
```
Comment (dhimanaishwarya) [inline CSS/documentation.css:88]: remove comments which are not required

File: components/avatar/avatar.html [hunk 5]
```diff
@@ -65,43 +68,85 @@ <h3>Components</h3>
...
+                        <section class="component-scetion mb-2">
```
Comment (dhimanaishwarya) [inline components/avatar/avatar.html:77]: typo in the class name , also if you are not using this class you can remove it

**Review by dhimanaishwarya — CHANGES_REQUESTED** [review]

**Comment by dhimanaishwarya** [comment]
UI looks good

File: CSS/documentation.css [hunk 6]
```diff
@@ -68,23 +69,47 @@ main{
...
+    font-family: var(--component-heading-font);
+}
+
+.content-main-section p{
+    font-size: 2rem;
```
Comment (rahulyadav139) [inline CSS/documentation.css:93]: everything is great, use rem instead of px.

**Review by rahulyadav139 — APPROVED** [review]

File: prism/prism.css [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+  box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.2);
+  border-radius: 0.5em;
+}
+div.code-toolbar > .toolbar > .toolbar-item > a:focus,
+div.code-toolbar > .toolbar > .toolbar-item > a:hover,
+div.code-toolbar > .toolbar > .toolbar-item > button:focus,
+div.code-toolbar > .toolbar > .toolbar-item > button:hover,
+div.code-toolbar > .toolbar > .toolbar-item > span:focus,
+div.code-toolbar > .toolbar > .toolbar-item > span:hover {
+  color: inherit;
```
Comment (dev-ashishsethi) [inline prism/prism.css:199]: looks good to me you can merge

**Review by dev-ashishsethi — APPROVED** [review]
looks good

**Review by j24m — APPROVED** [review]
Good work bro , code seems more than good and UI too. GTM!! :)

File: components/badges/badge.css [hunk 8]
```diff
@@ -1,3 +1,54 @@
...
+}
+
+.bottom{
+    bottom : 0;
+}
+
+.top{
+    top : -1rem;
+}
+
```
Comment (m26unkwn) [inline components/badges/badge.css:24]: This is too good to be true you are simplifying it.

**Review by m26unkwn — APPROVED** [review]
GTM

**Pull request merged**